tasks/vg_map_hts.wdl

```diff
@@ -148,3 +149,274 @@ task extractReference {
         docker: "quay.io/vgteam/vg:ci-684-bc9aa5dfc4b0d14519ea47333075906a4ec74656"
     }
 }
+
+task createDistanceIndex {
```

Comment:
Maybe all these indexing tasks want to be in a new file? They don't have all that much to do with `vg map` or with HTSlib (SAM/BAM/CRAM) files.